README.md

```diff
@@ -63,7 +63,8 @@ Who uses NATS? See our [list of users](https://nats.io/#who-uses-nats) on [https
 
 ### Security Audit
 
-A third party security audit was performed by Cure53, you can see the full report [here](https://github.com/nats-io/nats-general/blob/main/reports/Cure53_NATS_Audit.pdf).
+A third-party security audit was performed by ostif.org with the help of "Trail of Bits".  
+You can see the full report [on Trail of Bit’s Github](https://github.com/trailofbits/publications/blob/master/reviews/2025-04-ostif-nats-securityreview.pdf).
```

Comment:
"Trail of Bits GitHub", no apostrophe. Can just change the link to "here"?

Reply:
I personally like longer links because I think 4 chars are easier to miss, but I can change if you insist :-)

Reply:
I suggest linking to the blog post and the report directly. A compromise would be be two links that are named and have purpose.